### PR TITLE
[TACHYON-1714] Forget to set executor resources (cpu/memory)

### PR DESCRIPTION
**Pull request by BrickXu**
If we use a custom executor to run task, we should set cpus/mem for the executors, the minimum requires are 0.1/32MB.

In this PR, we set these resources by default value.

### CODE REVIEW

File: integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java [hunk 1]
```diff
@@ -274,6 +276,17 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
           break;
       }
     }
+
+    private List<Protos.Resource> getExecutorResources() {
+      List<Protos.Resource> resources = new ArrayList<Protos.Resource>(2);
+      resources.add(Protos.Resource.newBuilder().setName(Constants.MESOS_RESOURCE_CPUS)
+              .setType(Protos.Value.Type.SCALAR)
+              .setScalar(Protos.Value.Scalar.newBuilder().setValue(0.1d)).build());
```
Comment (jsimsa) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: please add a comment here that explains the meaning of the constant

File: integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java [hunk 2]
```diff
@@ -274,6 +276,17 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
...
+      resources.add(Protos.Resource.newBuilder().setName(Constants.MESOS_RESOURCE_MEM)
+              .setType(Protos.Value.Type.SCALAR)
+              .setScalar(Protos.Value.Scalar.newBuilder().setValue(32.0d)).build());
```
Comment (jsimsa) [inline integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java]: ditto

**Comment by jsimsa** [comment]
@BrickXu thank you for the fix, just one minor comment

**Comment by jsimsa** [comment]
ok to test

**Comment by BrickXu** [comment]
@jsimsa Thanks for reviewing :)

**Comment by jsimsa** [comment]
@BrickXu thanks for the update, left some comments

**Comment by BrickXu** [comment]
Hi  @jsimsa, thanks very very much for reviewing PR.

BTW, I left one comment in https://tachyon.atlassian.net/browse/TACHYON-1444, how about your idea about it ?

**Comment by jsimsa** [comment]
LGTM, am going to take a look at the JIRA right now

**Comment by yupeng9** [comment]
LGTM. Thanks for the improvement!

**Pull request merged**